README.md

```diff
@@ -1,5 +1,20 @@
 # CodeReady Dependency Analytics (crda)
 
+[![CI Checks](https://github.com/redhat-actions/crda/actions/workflows/ci.yml/badge.svg)](https://github.com/redhat-actions/crda/actions/workflows/ci.yml)
+[![Link checker](https://github.com/redhat-actions/crda/actions/workflows/link_checker.yml/badge.svg)](https://github.com/redhat-actions/crda/actions/workflows/link_checker.yml)
+<br>
+<br>
+[![Scan Golang project](https://github.com/redhat-actions/crda/actions/workflows/scan_go.yml/badge.svg)](https://github.com/redhat-actions/crda/actions/workflows/scan_go.yml)
+[![Scan Java project](https://github.com/redhat-actions/crda/actions/workflows/scan_java.yml/badge.svg)](https://github.com/redhat-actions/crda/actions/workflows/scan_java.yml)
+[![Scan Node projects](https://github.com/redhat-actions/crda/actions/workflows/scan_node.yml/badge.svg)](https://github.com/redhat-actions/crda/actions/workflows/scan_node.yml)
+[![Scan Node project with deps_install_cmd](https://github.com/redhat-actions/crda/actions/workflows/scan_custom_deps.yml/badge.svg)](https://github.com/redhat-actions/crda/actions/workflows/scan_custom_deps.yml)
+[![Scan Python project](https://github.com/redhat-actions/crda/actions/workflows/scan_python.yml/badge.svg)](https://github.com/redhat-actions/crda/actions/workflows/scan_python.yml)
```

Comment:
Let's replace `https://github.com/redhat-actions/crda` with `.` where we can (URLs to files)

Reply:
I think this will create problem in a PR, because in PR we are not running any workflow and when link checker run it will show 404